### PR TITLE
Feature/teacher profile finalize

### PR DESCRIPTION
**Pull request by sunkist5691**
Able to update teacher profile with associated `teacher id`
Able to clear all bugs/issues when loading teacher edit form
Able to keep teacher dashboard when moving around with teacher-related components

### CODE REVIEW

File: src/components/pages/Teacher/TeacherProfile/TeacherProfile.js [hunk 1]
```diff
@@ -17,42 +18,45 @@ const TeacherProfile = props => {
...
   return (
-    <Profile>
-      <ComponentTitle titleText="Village" />
-      <img src={`${profile.teachers_picture}`} />
-      <Label>Name:</Label>
-      <p>{`${profile.first_name} ${profile.last_name}`}</p>
+    <>
+      <TeacherDashboard />
+      <Profile>
+        <ComponentTitle titleText="Village" />
```
Comment (joshwhitwell) [inline src/components/pages/Teacher/TeacherProfile/TeacherProfile.js]: Should the title here be "Teacher" since it is the teacher Profile component?

**Review by joshwhitwell — APPROVED** [review]
Looks good, noticed that the teacher Profile title says "Village", might want to change to "Teacher".

**Review by andre-jeon — APPROVED** [review]
Looks good!

**Review by RohithSachdeva — APPROVED** [review]
Coming along nicely!

**Pull request merged**